Packs/FeedGithub/Integrations/FeedGithub/FeedGitHub.py

```diff
@@ -0,0 +1,603 @@
+import demistomock as demisto
+from CommonServerPython import *
+from TAXII2ApiModule import *
+import plyara
+import plyara.utils
+import tldextract
+
+CONTEXT_PREFIX = "GITHUB"
+RAW_RESPONSE = []
+
+
+class Client(BaseClient):
+    def __init__(self, base_url: str, verify: bool, proxy: bool, owner: str, repo: str, headers: dict):
+        base_url = urljoin(base_url, f"/repos/{owner}/{repo}")
+        super().__init__(base_url=base_url, verify=verify, proxy=proxy, headers=headers)
+
+    def get_commits_between_dates(self, since, until) -> list:
+        """
+        Retrieves the SHA of the base commit of the repository.
+
+        This function fetches all commits of the repository and returns the SHA of the latest commit,
+        which represents the base commit of the repository.
+
+        Returns:
+            str: The SHA of the base commit of the repository.
+        """
+        parsed_since = dateparser.parse(since)
+        parsed_until = dateparser.parse(until)
+        if not (parsed_since and parsed_until):
```

Comment:
if either of these arent provided lets just not send the param to the api. No need to enforce this if the api doesnt

Reply:
done